elasticsearch/example/src/test/java/example/springdata/elasticsearch/conference/ElasticsearchOperationsTest.java

```diff
@@ -73,6 +77,8 @@ public ClientConfiguration clientConfiguration() {
 	}
 
 	@Autowired ElasticsearchOperations operations;
+	@Autowired
```

Comment:
Why is this injecting the Template object? We have already `ElasticsearchOperations` at hand.